app/models/video.rb

```diff
@@ -0,0 +1,21 @@
+class Video < ApplicationRecord
+  has_many :rentals
+  has_many :customers, through: :rentals
+
+  validates :title, :overview, :release_date, :total_inventory, :available_inventory, presence: true
+
+  def decrease_inventory
+    self.available_inventory -= 1
+    self.save
+  end
+
+  def increase_inventory
+    self.available_inventory += 1
+    self.save
+  end
```

Comment:
This works, but I would suggest instead writing a method to check the available inventory by the number checked out and the number total, rather than make a separate field for available inventory.